src/components/pages/ConfirmOTP/ConfirmOTP.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import styles from './ConfirmOTP.module.css';
+import { Typography } from '@material-ui/core';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import { Button } from '../../UI/Form/Button/Button';
+import { REACT_APP_GLIFIC_CHAT_REDIRECT } from '../../../config/axios';
+import clsx from 'clsx';
+
+export interface ConfirmOTPProps {
+  location: any;
+}
+
+export const ConfirmOTP: React.SFC<ConfirmOTPProps> = (props) => {
+  const [userAuthCode, setuserAuthCode] = useState('');
+  const [correctAuthCode, setCorrectAuthCode] = useState(
+    props.location ? props.location.state.authCode : undefined
+  );
+
+  const handleuserAuthCodeChange = () => (event: React.ChangeEvent<HTMLInputElement>) => {
+    setuserAuthCode(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    if (userAuthCode == correctAuthCode) {
+      window.location.href = REACT_APP_GLIFIC_CHAT_REDIRECT;
```

Comment:
Is this method of redirecting the user to the chat page okay? Is there a better way to do it?

Reply:
You can use <Redirect> to send to chat page.